web/src/main/java/org/cbioportal/web/parameter/filter/SampleTreatmentFilter.java

```diff
@@ -0,0 +1,39 @@
+package org.cbioportal.web.parameter.filter;
+
+import org.cbioportal.model.TemporalRelation;
+import org.cbioportal.model.SampleTreatmentRow;
+import org.cbioportal.web.parameter.SampleIdentifier;
+
+import java.util.Map;
+
+public class SampleTreatmentFilter {
+    private String treatment;
+    private TemporalRelation time;
+
+    /**
+     * A sample should be included if the treatment row that corresponds
+     * to the treatment and time in this filter contains that sample.
+     * @param treatments key is SampleTreatmentRow::calculateKey
```

Comment:
Better to add the same comment `* @param sampleId sample.STABLE_ID` as `PatientTreatmentFilter.java`